test/integration/base_integration_test.h

```diff
@@ -208,10 +208,19 @@ class BaseIntegrationTest : protected Logger::Loggable<Logger::Id::testing> {
       const std::vector<std::string>& expected_resource_names, bool expect_node = false,
       const Protobuf::int32 expected_error_code = Grpc::Status::WellKnownGrpcStatus::Ok,
       const std::string& expected_error_message = "");
+  // For SotW tests, provides the same functionality as the function above, except it also takes in
+  // the FakeStream instead of using xds_stream_. Continue to use the above function for SotW tests
+  // unless the xDS test is not using xds_stream_.
+  AssertionResult compareSotwDiscoveryRequest(
```

Comment:
could we just extend the function above to take stream as a last argument and default it to xds_stream_ ?  Ditto below

Reply:
Good point.  Non-static members can't be used as default arguments, so I can't directly assign the default to xds_stream_, but I made the default nullptr and if it's nullptr, use xds_stream_.